internal/command/tokens/list.go

```diff
@@ -0,0 +1,77 @@
+package tokens
+
+import (
+	"context"
+	"fmt"
+	"github.com/spf13/cobra"
+	"github.com/superfly/flyctl/api"
+	"github.com/superfly/flyctl/client"
+	"github.com/superfly/flyctl/internal/appconfig"
+	"github.com/superfly/flyctl/internal/command"
+	"github.com/superfly/flyctl/internal/command/orgs"
+	"github.com/superfly/flyctl/internal/flag"
+	"github.com/superfly/flyctl/internal/render"
+	"github.com/superfly/flyctl/iostreams"
+)
+
+func newList() *cobra.Command {
+	const (
+		short = "List deploy tokens"
```

Comment:
We might have non-deploy tokens at some point

```suggestion
		short = "List tokens"
```